src/screens/SignIn/index.tsx

```diff
@@ -13,20 +19,38 @@ export const signInScreenTestIds = {
   passwordField: 'sign-in-form__input-password',
   forgotButton: 'sign-in-form__forgot-button',
   signInButton: 'sign-in-form__button',
+  errorAlert: 'sign-in__error-alert',
+  loadingDialog: 'sign-in__loading-dialog',
 };
 
 const SignInScreen = (): JSX.Element => {
-  const handleSubmit = async (event: React.SyntheticEvent) => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const { loading, errors, success } = useAppSelector((state) => state.auth);
+
+  const navigate = useNavigate();
+
+  const dispatch = useAppDispatch();
+
+  const handleSubmit = (event: React.SyntheticEvent) => {
```

Comment:
Later, you can check the [React Hook Form](https://www.react-hook-form.com/) to refactor forms. That lib can support validation, error messages,...

Reply:
Thanks, noted it 😍